### PR TITLE
Add config content variables for readability.

### PR DESCRIPTION
**Pull request by marcemq**
Changes proposed on the PR:
- Add `sentinelConfigFileNameContent`, `redisConfigFileNameContent` and `shutdownContent` to hold the proper content of each `CongifMap` data entry.

Signed-off-by: Morales Quispe, Marcela <marcela.morales.quispe@intel.com>

### CODE REVIEW

File: operator/redisfailover/service/generator.go [hunk 1]
```diff
@@ -88,6 +88,10 @@ func generateSentinelConfigMap(rf *redisfailoverv1alpha2.RedisFailover, labels m
 	namespace := rf.Namespace
 
 	labels = util.MergeLabels(labels, generateLabels(sentinelRoleName, rf.Name))
+	sentinelConfigFileNameContent := `sentinel monitor mymaster 127.0.0.1 6379 2
```
Comment (jchanam) [inline operator/redisfailover/service/generator.go]: I'd rename this to `sentinelConfigFileContent`

File: operator/redisfailover/service/generator.go [hunk 2]
```diff
@@ -110,6 +111,10 @@ func generateRedisConfigMap(rf *redisfailoverv1alpha2.RedisFailover, labels map[
 	namespace := rf.Namespace
 
 	labels = util.MergeLabels(labels, generateLabels(redisRoleName, rf.Name))
+	redisConfigFileNameContent := `slaveof 127.0.0.1 6379
```
Comment (jchanam) [inline operator/redisfailover/service/generator.go]: I'd rename this to `redisConfigFileContent`

File: operator/redisfailover/service/generator.go (hunk 1 above)
Comment (marcemq) [reply operator/redisfailover/service/generator.go]: I'll change it 😊

File: operator/redisfailover/service/generator.go (hunk 2 above)
Comment (marcemq) [reply operator/redisfailover/service/generator.go]: I'll change it 😊

**Comment by marcemq** [comment]
> Do you think moving the content of the file improves readability? Is there more motivations to move the content of the files to variables?
 Yes 😊

**Pull request merged**